gke-deploy.sh

```diff
@@ -17,7 +17,7 @@
 
 set -eu
 
-export TAG=2.16.1;
+export TAG=2.14.1;
```

Comment:
`2.14.1` should work here but be aware `2.14.0` is the image i submitted to marketplace

Reply:
Will update, thanks!